<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="top_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#36393e</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="content">
      <property name="styleSheet">
       <string notr="true">background-color: #424549</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="login_area">
         <property name="maximumSize">
          <size>
           <width>450</width>
           <height>550</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #F9FBFF;
border-radius: 5px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="logo">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>30</y>
            <width>241</width>
            <height>311</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image:url(:/logo_icon/images/logo_icon.png) 0 0 0 0 stretch stretch;
background-repeat: no-repeat;
background-position: center;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLineEdit" name="user_line_edit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>350</y>
            <width>271</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	font: 75 12pt &quot;Verdana&quot;;
	border: 2px solid rgb(49, 87, 121);
	border-radius: 5px;
	padding: 15px;
	padding-top: 5px;
	padding-bottom: 5px; 
}

QLineEdit:hover {
	border: 2px solid rgb(54, 54, 54)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>USER</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="password_line_edit">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>400</y>
            <width>271</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	font: 75 12pt &quot;Verdana&quot;;
	border: 2px solid rgb(49, 87, 121);
	border-radius: 5px;
	padding: 15px;
	padding-top: 5px;
	padding-bottom: 5px; 
}

QLineEdit:hover {
	border: 2px solid rgb(54, 54, 54)
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>PASSWORD</string>
          </property>
         </widget>
         <widget class="QPushButton" name="login_button">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>460</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(49, 87, 121);
	font: 75 12pt &quot;Verdana&quot;;
	color: white;
}

QPushButton:hover {
	background-color:rgb(61, 108, 150)
}</string>
          </property>
          <property name="text">
           <string>LOGIN</string>
          </property>
         </widget>
         <widget class="QPushButton" name="register_button">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>460</y>
            <width>141</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(49, 87, 121);
	font: 75 12pt &quot;Verdana&quot;;
	color: white;
}

QPushButton:hover {
	background-color:rgb(61, 108, 150)
}</string>
          </property>
          <property name="text">
           <string>REGISTER</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom_bar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>35</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #36393e</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="files_rc.qrc"/>
 </resources>
 <connections/>
</ui>
